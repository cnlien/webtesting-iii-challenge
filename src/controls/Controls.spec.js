import React from 'react';
import * as rtl from 'react-testing-library';
import 'jest-dom/extend-expect';
import Controls from './Controls';
afterEach(rtl.cleanup);

describe ('should match snapshot', () => {
    it('defaults to unlocked and open gate', () => {
        const controls = rtl.render(<Controls />);
        expect(controls.baseElement).toMatchSnapshot();
    });
});