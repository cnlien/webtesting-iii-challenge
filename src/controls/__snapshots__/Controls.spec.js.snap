// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot defaults to unlocked and open gate 1`] = `
<body>
  <div>
    <div
      class="controls panel"
    >
      <button
        class="toggle-btn"
        disabled=""
      >
        Lock Gate
      </button>
      <button
        class="toggle-btn"
      >
        Close Gate
      </button>
    </div>
  </div>
</body>
`;
